### PR TITLE
New operator release 2.9.0

### PR DESCRIPTION
**Pull request by Nidhi251289**
new operator releaser with release notes and updated documentation

### CODE REVIEW

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md [hunk 1]
```diff
@@ -770,7 +785,7 @@ For **Google Cloud Container Registry**, the supported authentication is [worklo
 * `GCP Service Account`: [google service account](https://cloud.google.com/iam/docs/service-accounts) — for example `service-account-name@project-id.iam.gserviceaccount.com`.
 * `Kubernetes Service Account`: the kubernetes service account that will be created and annotated with your google service account during post configuration. You need to [bind](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#authenticating_to) the kubernetes service account to your google service account.
 
-    Below is an example how to bind a google cloud service account to a kubernetes service account:
```
Comment (zlogic) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md:781]: Can you revert this change? I think this might break the formatting in the document

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md [hunk 2]
```diff
@@ -348,10 +348,19 @@ To use this plan, [upgrade](/developerportal/deploy/private-cloud-upgrade-guide/
...
+* Amazon S3
+* Azure Blob Storage
+* Google Cloud Storage bucket
+* Ceph RADOS
+
+**MinIO** will connect to a [MinIO](https://min.io/product/overview) S3-compatible object storage. You will need to provide all the information about your MinIO storage such as endpoint, access key, and secret key. The MinIO server needs to be a full-featured MinIO server, or a [MinIO Gateway](https://github.com/MinIO/MinIO/tree/master/docs/gateway) with configured etcd.
 
 {{% alert color="info" %}}
-To use TLS, specify the MinIO URL with an `https` schema, for example `https://minio.local:9000`. If MinIO has a self-signed certificate, you'll also need to configure [custom TLS](#custom-tls) so that the self-signed certificate is accepted.
+To use TLS, specify the MinIO URL with an `https` schema, for example `https://MinIO.local:9000`. If MinIO has a self-signed certificate, you'll also need to configure [custom TLS](#custom-tls) so that the self-signed certificate is accepted.
```
Comment (zlogic) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: Can you keep the original URL (`https://minio.local:9000`)?

Comment (Nidhi251289) [reply content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: ok

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md (hunk 1 above)
Comment (Nidhi251289) [reply content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md:781]: done

**Comment by MarkvanMents** [comment]
Hi Nidhi - you seem to be branching off a very old fork of development, so we are getting conflicts.
This is also the case for #5269
I can't easily sort out the conflicts when they are on your branch, so I'll need to wait until we are ready to merge and then do it.

Can you see if you can bring your development fork up to date before you make the next PR so we can avoid this next time?
Let me know if you have any questions.
Thanks.

**Comment by Nidhi251289** [comment]
HI Mark,

Thank you informing this issue on my side.

I will check on my end, why i am not getting the latest branch. Meanwhile, we are planning to release the new operator version today, can you please review the PR. Once we are done with release, can you also publish the documentation. Kindly change the date in the operator release to today.

Regards     
nidhi
________________________________

**Comment by MarkvanMents** [comment]
Operator released.

**Review by MarkvanMents — APPROVED** [review]
